stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/list/Messages.kt

```diff
@@ -284,7 +279,7 @@ internal fun BoxScope.DefaultMessagesHelperContent(
             newMessageState,
             areNewestMessagesLoaded,
             lazyListState.isScrollInProgress,
-        ) || hasLoadedThread
+        )
```

Comment:
`hasLoadedThread` was always turning `shouldScrollToBottom` into `true` value, that's was forcing Thread Replies to be scrolled to the bottom:
```kotlin
val shouldScrollToBottom = shouldScrollToBottom(
    // ...
) || hasLoadedThread
```

In addition, it's not clear why we mix `shouldScrollToBottom` with `hasLoadedThread`, because `hasLoadedThread` is for threads use-case only, however, we use `DefaultMessagesHelperContent` component for both messages and threads